blog/2018-07-17-UnitTestsInXslxFileForConferenceScheduling.adoc

```diff
@@ -0,0 +1,122 @@
+= Unit tests in .xlsx files
```

Comment:
The title must catch your attention like a newspaper headline. The title is not the time to be complete or comprehensive. 

What is the main benefit of this article to readers? I'd say: to allow business people to compose and validate unit tests.

Proposals for the title:
A) Let the business users write the JUnit tests in Excel or LibreOffice
B) Unit testing constraints with business input from Excel/LibreOffice

The adoc file name needs to be adjusted accordingly.